examples/default/versions.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_version = ">= 1.0.0"
+}
+
+provider "azurerm" {
+  features {}
+}
+
+provider "azuread" {}
```

Comment:
Is this provider block needed?

Reply:
No it is not. Thanks for flagging. Removed.